apps/explorer/src/ui/VerticalList.tsx

```diff
@@ -16,9 +16,9 @@ export function ListItem({ active, children, onClick }: ListItemProps) {
             <button
                 type="button"
                 className={clsx(
-                    'cursor-pointer px-3 py-2 rounded-md text-body block w-full border-1 border-solid text-left',
+                    'cursor-pointer py-2 rounded-md text-body block w-full text-left mt-0.5 px-1.5 border',
```

Comment:
These shouldn't have top margin, the composing parent should add margin.

Also in the design the x padding looks like it's 10px (2.5), not sure why it's adjusted to 1.5.